promotion/datastore.go

```diff
@@ -691,3 +703,80 @@ limit 1`
 
 	return attempted, nil
 }
+
+// This code can be deleted once https://github.com/brave-intl/bat-go/issues/263 is addressed.
+
+// GetOrder queries the database and returns an order
+func (pg *Postgres) GetOrder(orderID uuid.UUID) (*Order, error) {
+	statement := "SELECT * FROM orders WHERE id = $1"
+	order := Order{}
+	err := pg.DB.Get(&order, statement, orderID)
+	if err == sql.ErrNoRows {
+		return nil, nil
+	} else if err != nil {
+		return nil, err
```

Comment:
Might be good to wrap these errors before they are returned?  `fmt.Errorf("Failed to get order : %w", err)` for example to get some context and have a stack of errors for the caller?

Reply:
https://github.com/brave-intl/bat-go/pull/264/commits/afba0039ca66688d6f6675850e093e1636b5a2a4